db/migrate/20231009185513_create_locations.rb

```diff
@@ -0,0 +1,14 @@
+class CreateLocations < ActiveRecord::Migration[7.0]
+  def change
+    create_table :locations do |t|
+      t.inet :ip_address, null: false, index: { unique: true }
+      t.string :country_code, limit: 2, null: false
+      t.string :country, limit: 100, null: false
+      t.string :city, limit: 100, null: false
+      t.float :latitude, null: false
+      t.float :longitude, null: false
+
+      t.timestamps
```

Comment:
* I didn't have enough time to use `rom-rb` for migrations, so I did it via `ActiveRecord`
* Postgres supports `inet` type to optimally store IP-addresses. This column also has a unique index to ensure that there are no 2 records with the same IP-address and speed up the lookup by ip-address that we use in the HTTP endpoint